src/lib/configUtils.ts

```diff
@@ -28,9 +28,20 @@ function isPublic(rule: APIConfigRule): boolean {
   return rule.conditions[0].type === 'public';
 }
 
-export function getStaleConfig(configName: string): StatsigConfig[] {
+export function getStaleConfigWithReason(
```

Comment:
Returns an array, but is called `getStaleConfig` - either return a single result, or call it `getStaleConfigs`

Seems like it just needs to return a single result

Reply:
Yeah might rename to something like `getStaleConfigByProject`